app/views/checkout/edit.html.haml

```diff
@@ -23,8 +23,21 @@
     = #TODO create new component ? reuse out_of_stock.html.haml template
     - if @any_out_of_stock
       = render ModalComponent.new(id: "out-of-stock-items", instant: true) do
-        %div
-          Some items are out of stock
+        %h3
+          = t("js.out_of_stock.reduced_stock_available")
+        %p
+          = t("js.out_of_stock.out_of_stock_text")
+        %p
+          - @updated_variants.each do |variant|
+            %em 
+              = "#{variant.name_to_display} - #{variant.unit_to_display}"
+              - if variant.on_hand == 0
+                %span
+                  = t("js.out_of_stock.now_out_of_stock")
+              - if variant.on_hand > 0
+                %span
+                  = t("js.out_of_stock.only_n_remainging", num: variant.on_hand)
```

Comment:
Typo? Oh no, the typo is in the translation files!

I would either add a comment `# sic!` here, rename it or maybe start new lazy translations so that we can delete the old ones when we get rid of AngularJS.

Reply:
Never mind, I see you've resolved this already.